mlos_bench/mlos_bench/storage/sql_schema.py

```diff
@@ -0,0 +1,143 @@
+            Column("config_id", Integer, nullable=False),
+            Column("ts_start", DateTime, nullable=False, default="now"),
+            Column("ts_end", DateTime),
+            # Should match the text IDs of `mlos_bench.environment.Status` enum:
+            Column("status", String(16), nullable=False),
+
+            PrimaryKeyConstraint("exp_id", "trial_id"),
+            ForeignKeyConstraint(["exp_id"], [self.experiment.c.exp_id]),
+            ForeignKeyConstraint(["config_id"], [self.config.c.config_id]),
+        )
+
+        # Values of the tunable parameters of the experiment,
+        # fixed for a particular trial config.
+        self.config_param = Table(
+            "config_param",
+            self._meta,
+            Column("config_id", Integer, nullable=False),
+            Column("param_id", String(255), nullable=False),
+            Column("param_value", String(255)),
+
+            PrimaryKeyConstraint("config_id", "param_id"),
+            ForeignKeyConstraint(["config_id"], [self.config.c.config_id]),
+        )
+
+        # Values of additional non-tunable parameters of the trial,
+        # e.g., scheduled execution time, VM name / location, number of repeats, etc.
+        self.trial_param = Table(
+            "trial_param",
+            self._meta,
+            Column("exp_id", String(255), nullable=False),
+            Column("trial_id", Integer, nullable=False),
+            Column("param_id", String(255), nullable=False),
+            Column("param_value", String(255)),
+
+            PrimaryKeyConstraint("exp_id", "trial_id", "param_id"),
+            ForeignKeyConstraint(["exp_id", "trial_id"],
+                                 [self.trial.c.exp_id, self.trial.c.trial_id]),
+        )
+
+        self.trial_result = Table(
+            "trial_result",
+            self._meta,
+            Column("exp_id", String(255), nullable=False),
+            Column("trial_id", Integer, nullable=False),
+            Column("metric_id", String(255), nullable=False),
+            Column("metric_value", String(255)),
+
+            PrimaryKeyConstraint("exp_id", "trial_id", "metric_id"),
+            ForeignKeyConstraint(["exp_id", "trial_id"],
+                                 [self.trial.c.exp_id, self.trial.c.trial_id]),
+        )
+
+        self.trial_telemetry = Table(
+            "trial_telemetry",
+            self._meta,
+            Column("exp_id", String(255), nullable=False),
+            Column("trial_id", Integer, nullable=False),
+            Column("ts", DateTime, nullable=False, default="now"),
+            Column("metric_id", String(255), nullable=False),
+            Column("metric_value", String(255)),
+
+            UniqueConstraint("exp_id", "trial_id", "ts", "metric_id"),
+            ForeignKeyConstraint(["exp_id", "trial_id"],
+                                 [self.trial.c.exp_id, self.trial.c.trial_id]),
+        )
+
+        _LOG.debug("Schema: %s", self._meta)
+
+    def create(self):
```

Comment:
Request to add a `dump()` function so that we could do something like this to get the raw schema sql text for debugging:
```sh
python3
>> ...
```
```python
import mlos_bench.mlos_bench.storage.sql_schema as sql_schema
print(sql_schema.dump())
```

Reply:
yep. will add it now